bigquery_etl/cli/query.py

```diff
@@ -976,8 +981,22 @@ def _run_query(
             )
             query_stream.seek(0)
 
-            # run the query as shell command so that passed parameters can be used as is
-            subprocess.check_call(["bq"] + query_arguments, stdin=query_stream)
+            if mapped_values is None or parallelized is False:
+                # Run the query as shell command so that passed parameters can be used as is.
+                subprocess.check_call(["bq"] + query_arguments, stdin=query_stream)
+            else:
+                query_content = query_stream.read()
+                query = query_content.format(
+                    mapped_values=mapped_values,
+                )
```

Comment:
I am hesitant about making any changes to the `_run_query` function; it shouldn't have to know if this is a parallel execution. The mapped values can either be transmitted into the function via query parameters, or jinja templating (but the former will probably be simpler). If we do that we can remove this section and use the subprocess call for all uses of `_run_query`.

Reply:
Makes sense. The change in run_query() is limited to determine if the mapped_values should be added to the query arguments.